### DISCUSSION TITLE
The sunbird API documention link (http://docs.sunbird.org/latest/apis) is not working

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#607] [General Discussion]
Hello all,

The sunbirdEd API documentation link (http://docs.sunbird.org/latest/apis) is not working. Throwing **404 error**.
Is it down for some reason or is it removed?

@rajeevsathish @Krishnaj20

**Comment by gandham-santhosh** [comment]
@nidhikachhawa24 docs.sunbird.org is decommissioned. Please check sunbird.org or ed.sunbird.org if you are referring Ed building block documentation. 
We applied URL forwarding rule for docs.sunbird.org to redirect to sunbird.org. But, looks like its not working for URI's. We will look into it
